proto/tendermint/abci/types.proto

```diff
@@ -37,6 +36,7 @@ message Request {
     RequestLoadSnapshotChunk  load_snapshot_chunk  = 14;
     RequestApplySnapshotChunk apply_snapshot_chunk = 15;
   }
+  reserved = 4;
```

Comment:
Nice reserving the value!
Not sure the '=' is valid syntax (as the linter points out)

Reply:
The reserve should be formatted correctly now